windows/hardening/AD-Hardening.ps1

```diff
@@ -259,10 +282,23 @@ function Configure-Firewall {
     try {
         :outer while ($true) {
             $desigPorts = Get-Comma-Separated-List -message "List needed port numbers for firewall config. Separate by commas."
-            $usualPorts = @(53, 3389, 80, 445, 139, 22, 88)
-
+            $usualPorts = @(53, 3389, 80, 445, 139, 22, 88, 67, 68, 135, 139, 389, 636, 3268, 3269, 464) | Sort-Object
+            Write-Host "All the following ports that we suggest are either common scored services, or usually needed for AD processes. We will say which is which"
             foreach ($item in $usualPorts) {
-                if ($ports -notcontains $item) {
+                if ($desigPorts -notcontains $item) {
+                    if ($item -in @(53, 3389, 80, 22)) {
+                        Write-Host "`nCommon Scored Service" -ForegroundColor Green
+                    }
+                    if ($item -in @(139, 88, 67, 68, 135, 139, 389, 445, 636, 3268, 3269, 464)) {
```

Comment:
looks like you could use the variable $usualPorts here.

Reply:
I didn't exactly do that, but I made it cleaner